app/1.0/docs/tools/advanced.md

```diff
@@ -133,20 +133,23 @@ Usage:
 
 Source: [github.com/PolymerLabs/polyclean](https://github.com/PolymerLabs/polyclean)
 
-### <b>polybuild</b>—all-in-one build tool for optimizing apps {#polybuild}
+### <b>polymer-build</b> — Provides granular control of building and optimizing applications {#polymer-build}
```

Comment:
For consistency with the other headings, please remove the spaces around the em dash and lowercase "provides."